app/serializers/v2/moves_serializer.rb

```diff
@@ -39,14 +38,12 @@ class MovesSerializer
       to_location
       prison_transfer_reason
       supplier
-      allocation
     ].freeze
 
     belongs_to :from_location, serializer: ::LocationSerializer
     belongs_to :to_location, serializer: ::LocationSerializer
     belongs_to :profile, serializer: V2::ProfilesSerializer
     belongs_to :prison_transfer_reason, serializer: PrisonTransferReasonSerializer
     belongs_to :supplier, serializer: SupplierSerializer
-    belongs_to :allocation, serializer: AllocationSerializer
```

Comment:
I'm wondering whether it's useful to leave this relationship in place so that clients could potentially make a separate API call for the relevant individual allocation 🤔

Reply:
Does this remove the relationship completely? I think it would be good to preserve the relationships but agree that we can remove the include.

In my mind, I think it's easier to understand if the resource structure is the same both when asking for 1 or many.

If we change the structure (attributes/relationships) I've found it can lead to confusion when trying to develop.

Reply:
Yes, I think I agree. The danger is including an allocation, but referencing the allocation relationship doesn't have any performance overhead as we already have the allocation_id available on the move when serializing. I'll reinstate this relationship.